gateway/gateway.go

```diff
@@ -551,15 +553,18 @@ func (gateway *HandleT) webRequestRouter() {
 func (gateway *HandleT) printStats() {
 	for {
 		time.Sleep(10 * time.Second)
-		logger.Info("Gateway Recv/Ack ", gateway.recvCount, gateway.ackCount)
+		recvCount := atomic.LoadUint64(&gateway.recvCount)
+		ackCount := atomic.LoadUint64(&gateway.ackCount)
+		logger.Info("Gateway Recv/Ack ", recvCount, ackCount)
```

Comment:
can you check here also once? https://github.com/rudderlabs/rudder-server-enterprise/blob/master/webhook/webhook.go#L277-L287